CONTRIBUTING.md

```diff
@@ -262,9 +321,11 @@ While the authors are no fans of blind adherence to style and so-called project
 that go through and correct code style, MetPy has adopted this style from the outset.
 Therefore, it makes sense to enforce this style as code is added to keep everything clean and
 uniform. To this end, part of the automated testing for MetPy checks style. To check style
-locally within the source directory you can use the ``flake8`` tool. Running it
-from the root of the source directory is as easy as running ``pytest --flake8`` in the base
-of the repository.
+locally within the source directory you can use the ``flake8`` tool.
+After setting up your [development environment](#setting-up-your-development-environment) above,
+install the code style tools we use with ``conda install --file ci/linting_requirements.txt``
+from within the ``metpy`` directory.
+Checking your code style is then as easy as running ``pytest --flake8`` in the base of the repository.
```

Comment:
Actually, none of those requirement files installs `pytest-flake8`. Best to document this as instead run `flake8 .` from the base of the repository.

Reply:
Sneaky. Very good catch.